_posts/2016-4-20-The Fashion Logging System -- ELK - Part 1.md

```diff
@@ -0,0 +1,161 @@
+# The Fashion Logging System -- ELK - Part 1
+
+## 前言：
+在做雲端服務的時候，有一個非常重要的環節，那就是 logging system！！  
+因為沒有 logging system 來幫你蒐集 log 並讓你容易 search 的話，先不論發生 bug 的時候不知道要從哪邊開始找起，就連伺服器掛掉狂噴 error 的時候，有可能客戶都會比你早發現，到時候就...  
+在威脅完大家 logging system 的重要性之後，這系列文章主要會分成以下三個部分去說明，讓大家對於 ELK 這套 logging system 有大致上的了解，並且可以架設出 ELK:
+1. ***什麼是 ELK ？***
```

Comment:
Add a space line above to make markdown compiler create an ordered list. (Check the [preview](https://github.com/iCHEF/ichef.github.io/blob/Alvin_ELK_Log_system/_posts/2016-4-20-The%20Fashion%20Logging%20System%20--%20ELK%20-%20Part%201.md))